tests/D2L.CodeStyle.TestAnalyzers.Tests/packages.config

```diff
@@ -6,7 +6,6 @@
   <package id="Microsoft.CodeAnalysis.CSharp.Workspaces" version="1.0.0" targetFramework="net461" />
   <package id="Microsoft.CodeAnalysis.Workspaces.Common" version="1.0.0" targetFramework="net461" />
   <package id="Microsoft.Composition" version="1.0.27" targetFramework="net461" />
-  <package id="NUnit" version="2.6.4" targetFramework="net461" />

```

Comment:
hmmm, wouldn't let me compile. Will remove these from the pull request once I'm done